pkg/util/kubernetes/kubelet/metadata.go

```diff
@@ -0,0 +1,43 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build kubelet
+
+package kubelet
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/DataDog/datadog-agent/pkg/metadata/host/container"
+)
+
+func init() {
+	container.RegisterMetadataProvider("kubelet", getMetadata)
+}
+
+func getMetadata() (map[string]string, error) {
```

Comment:
Are we planning to have more metadata from the kubelet beside the version ? If so  we should return a struct instead of a map. If not we could just return a string.

Reply:
So it's part of the metadata provider interface https://github.com/DataDog/datadog-agent/blob/master/pkg/metadata/host/container/loader.go#L25

The idea is that it's a map of KV metadatas.

Reply:
Ok, my bad !